Makefile

```diff
@@ -198,7 +199,10 @@ system-checks:
 	if [[ -z "$(GO)" ]]; then echo 'Missing "go" command.'; exit 1; fi
 
 require-s3cmd:
-	@if [[ -z "$(S3CMD)" ]]; then echo 'Missing "s3cmd" command.. See https://github.com/s3tools/s3cmd/blob/master/INSTALL'; exit 1; fi
+	@if [[ -z "$(S3CMD)" ]]; then echo 'Missing "s3cmd" command. See https://github.com/s3tools/s3cmd/blob/master/INSTALL'; exit 1; fi
```

Comment:
Not related to this PR, but I would say:

Missing "s3cmd" command. Run brew install s3cmd or see https://github.com/s3tools/s3cmd/blob/master/INSTALL